patches/chrome-installer-linux-common-rpmrepo.cron.patch

```diff
@@ -1,11 +1,20 @@
 diff --git a/chrome/installer/linux/common/rpmrepo.cron b/chrome/installer/linux/common/rpmrepo.cron
-index f7fe2bcf7d7cbf95b23067f21f87422706c5e4d0..3541826a03009e2adb8dbd6c258d254d31247a77 100755
+index f7fe2bcf7d7cbf95b23067f21f87422706c5e4d0..d0e47797a29882103c59b83a407d1932a006e93f 100755
 --- a/chrome/installer/linux/common/rpmrepo.cron
 +++ b/chrome/installer/linux/common/rpmrepo.cron
-@@ -14,6 +14,9 @@
+@@ -14,6 +14,18 @@
  # setting "repo_add_once" to "true" or "false" as desired. An empty
  # $DEFAULTS_FILE is the same as setting the value to "false".
- 
+
++# Remove Chrome repo if erroneously added (brave/brave-browser#2927)
++if [ -e "/etc/yum.repos.d/brave-browser.repo" ] ; then
```

Comment:
I tested this out and on Fedora the repository names here are unique for dev/beta, so this will only remove the release channel repo, but not remove repos previously created named `brave-browser-dev.repo` and `brave-browser-beta.repo`.

Reply:
Great catch! I had forgotten to build and test the new beta and dev packages.

I've verified my fix using the `beta` packages.